src/pages/use-less-energy/index.js

```diff
@@ -65,7 +65,7 @@ const UseLessEnergyPage = () => (
           <div className={ChoiceStyles.bannerActions}>
             <a
               className={ChoiceStyles.bannerCtaPrimary}
-              href="#actions"
+              href="use-less-energy/#actions"
```

Comment:
The same here: this reloads the page, but its also broken at the mo. Can we please remove the `use-less-energy/` and also remove the # from the associated id please :)